### PR TITLE
Adding Styled Table to container components

### PR DESCRIPTION
**Pull request by BrokenLadder**

### CODE REVIEW

File: packages/es-components/src/components/containers/table/table.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
+## Table
+Use `Table`, `TableRow`, `TableBodyCell` and `TableHeaderCell` components to create a styled table.
```
Comment (jrios) [inline packages/es-components/src/components/containers/table/table.md]: Delete until line 27.

File: packages/es-components/src/components/containers/table/table.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+    text-align: left;
+    `;
+
+    export const TableHeaderCell = TableBodyCell.withComponent('th');
+```
+
+## Basic Example
+
+```
+<Table>
```
Comment (jrios) [inline packages/es-components/src/components/containers/table/table.md]: This needs some actual content

**Comment by jrios** [comment]
make sure to add these to the [index.js](https://github.com/WTW-IM/es-components/blob/master/packages/es-components/src/index.js) file in `es-components`.

File: packages/es-components/src/index.js [hunk 3]
```diff
@@ -10,6 +10,10 @@ export Menu from './components/containers/menu/Menu';
 export TabPanel from './components/containers/tabPanels/TabPanel';
 export StripedContainer from './components/containers/striped-container/StripedContainer';
 export CheckboxLabel from './components/containers/checkboxLabel/CheckboxLabel';
+export Table from './components/containers/Table';
```
Comment (Darrken) [inline packages/es-components/src/index.js]: These imports need to be from `./components/containers/table/Table`

Comment (jrios) [reply packages/es-components/src/index.js]: I've got it taken care of on my end. 👍

Comment (Darrken) [reply packages/es-components/src/index.js]: What do you think about namespacing the sub elements so it's `Table.Row`, `Table.Cell`, etc.? That way you just have the one default Table export as well.

Comment (jrios) [reply packages/es-components/src/index.js]: I didn't think about it at the time. I will do that for the `next` release. 👍

**Pull request closed without merging**